src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  src: url(/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy", sans-serif;
+  font-size: 16px;
+  font-weight: 900;
+  color: #3d4e61;
+  src: url(/fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.search-bar {
+  position: relative;
+  display: block;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.search-bar:last-child {
+  margin: 0;
+}
+
+.labelStyle {
+  display: block;
+}
+
+.input-style {
+  box-sizing: border-box;
+  font-family: inherit;
+  outline: none;
+  height: 70px;
+  width: 100%;
+  font-size: 16px;
+  font-weight: 400;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background-color: #fff;
+}
+
+.smallSearchInput {
+  font-size: 14px;
+  height: 42px;
+  padding-left: 33px;
+}
+
+.loupe {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  z-index: 1;
+}
+
+.loupe--small {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.placeholder-style::placeholder {
+  font-family: inherit;
+  color: #3d4e61;
+}
+
+.hover-style:hover {
+  font-family: inherit;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.focus-style:focus {
+  font-family: "Avenir-Heavy", sans-serif;
+  color: #3d4e61;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
```

Comment:
No need to create separate class for each input state, you can set this styles under main class using pseudo-classes :focus, :hover and so on

Fix it everywhere